packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -251,3 +251,86 @@ class VelocityTracker {
     return Velocity(pixelsPerSecond: estimate.pixelsPerSecond);
   }
 }
+
+/// A [VelocityTracker] subclass that matches iOS scroll view's fling velocity
+/// calculation.
```

Comment:
Maybe briefly describe what's special about iOS scroll velocity to help bootstrap the next maintainer

Reply:
It's still not too clear what the difference is and why this needs to be a separate class rather than changing VelocityTracker.

Reply:
You mean the difference in terms of the strategies? Does
> /// A [VelocityTracker] subclass that employs a weighted average strategy to
/// estimate the velocity of the associated pointer, to match iOS scroll view's
/// fling velocity calculation.

sound better?

Reply:
As opposed to what though. I was looking for a text that saves the next maintainer from seeing thing as redundant, try to merge the 2 classes into one, spend 2 hours and then realized that the approach for the 2 are conceptually different (which the maintainer could have discovered in 10s by reading this text)

Reply:
Added a simple comparison of the 2 velocity tracker classes.